CONTRIBUTING.rst

```diff
@@ -566,6 +566,34 @@ Other Commit Expectations
   proposals, progress snapshots of your work, or to get early feedback on
   features or changes that will affect multiple areas in the code base.
 
+* When major new functionality is added, tests for the new functionality MUST be
+  added to the automated test suite. All new APIs MUST be documented and tested
+  and test MUST cover at least 80% of the added functionality using the code
+  coverage tool and reporting provided by the project.
+
+Submitting Proposals
+====================
+
+You can request a new feature or submit a proposal by submitting an issue to
+our GitHub Repository.
+If you would like to implement a new feature, please submit an issue with a
+proposal (RFC) for your work first, to be sure that we can use it. Please
```

Comment:
maybe mention adding ``RFC:`` to the title of the issue?

Reply:
detailed howto for proposal will follow.